pkg/controller/controller.go

```diff
@@ -203,6 +232,28 @@ func (p *csiProvisioner) Provision(options controller.VolumeOptions) (*v1.Persis
 	capacity := options.PVC.Spec.Resources.Requests[v1.ResourceName(v1.ResourceStorage)]
 	volSizeBytes := capacity.Value()
 
+	userCredentials := map[string]string{}
+	storageClassName := options.PVC.Spec.StorageClassName
+	fmt.Printf("><SB> provisioner was called for storage class: %s\n", *storageClassName)
```

Comment:
Remember to remove fmt.Printf statements

Reply:
Thanks, will do.